mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -0,0 +1,47 @@
+
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+from typing import Dict, List, Tuple
+
+from mitosheet.code_chunks.code_chunk import CodeChunk
+from mitosheet.excel_utils import get_df_name_as_valid_sheet_name
+from mitosheet.state import State
+from mitosheet.types import ColumnID
+
+
+class ExportToFileCodeChunk(CodeChunk):
+
+    def __init__(self, prev_state: State, post_state: State, _type: str, file_name: str, sheet_index_to_export_location: Dict[int, str]):
+        super().__init__(prev_state, post_state)
+        self._type = _type
```

Comment:
maybe we could use `export_type` so we don't need to use the underscore which is different from our other params